content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview and Terminology
+description: Learn about Pilot, Istio's core traffic management component, Envoy proxies, and how they enable service discovery and load balancing.
+weight: 1
+keywords: [traffic-management, pilot, envoy-proxies, service-discovery, load-balancing]
+---
+
+With Istio, you can manage [traffic routing](/docs/concepts/traffic-management/routing)
+and [load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+for your service mesh without having to update your services. Istio simplifies
+configuration of service-level properties like timeouts and retries, and makes
+it straightforward to set up tasks like staged rollouts with percentage-based
+traffic splits.
+
+Istio's traffic management model relies on the following two components:
+
+-  Pilot, the core traffic management component.
+-  Envoy proxies, which enforce configurations and policies set through Pilot.
+
+These components enable the following high-level tasks:
+
+-  Service discovery
+-  Load balancing
+
+## Pilot: Core traffic management {#pilot}
+
+The following diagram shows the Pilot architecture:
+
+{{< image width="40%"
+    link="./pilot-arch.svg"
+    caption="Pilot architecture"
+    >}}
+
+As the diagram illustrates, Pilot maintains an **abstract model** of all the
```

Comment:
Does it matter that it's abstract?

Reply:
Yes, because that signals to readers that it is not platform-specific.